### DISCUSSION TITLE
Certificates & secrets

### DISCUSSION THREAD

**Discussion by asa1253** [spencerwooo/onedrive-vercel-index#37] [Q&A] [answered]
Open Certificates & secrets panel, click New client secret and create a new secret called client_secret, set Expires to Never, click Add, and copy the Value of the client_secret (You only have this one chance to copy it.).

这一步没有Never选项了

**Comment by spencerwooo** [comment, marked as answer]
那就……选个最大的吧，我记得好像确实没有 never 的选项了，但是有一年（还是两年）的选项？

**Comment by xmxs** [comment]
拦截请求，然后编辑参数重发就可以了。可以轻松弄一个2099年到期的client secret。

**Reply by asa1253** [reply]
不会   我都是看着教程来的   到最后获取api哪里失败了  err什么的错误

**Reply by asa1253** [reply]
老师能出个教程么😁😁😁

**Reply by spencerwooo** [reply]
> 不会 我都是看着教程来的 到最后获取api哪里失败了 err什么的错误

这个错误跟你 `client_secret` 的 expire date 没啥关系，我猜测是你命令行工具用的不对。

**Reply by asa1253** [reply]
en  那个获取长期拦截请求那个呢    能试试出个教程么

**Reply by asa1253** [reply]
![error](https://100502.oss-cn-hangzhou.aliyuncs.com/2021-07-27_23-19.png?Expires=1628157267&OSSAccessKeyId=TMP.3KjTZT4GMHqz27Sas3KFw7BeMuihS7DLgcFZKJAvFAcAjfz6eXCRMH8D3fzAUN2n8yd5k4EGvZSejjPnFqepQ9DpHkbdDn&Signature=u8%2Bvqd7uVjSbfmU%2BdLqzRMq%2FgNU%3D)

**Reply by Kutius** [reply]
> 拦截请求，然后编辑参数重发就可以了。可以轻松弄一个2099年到期的client secret。

需要抓包嘛
